src/lib/exec-promise.js

```diff
@@ -4,23 +4,23 @@ const log = require('../lib/log');
 // TODO might be important to sanitise input to `exec()` to prevent injection
 // attacks by devious tech leads.
 
-module.exports = (...args) => new Promise((resolve, reject) => {
+module.exports = (log_level, ...args) => new Promise((resolve, reject) => {
```

Comment:
Oof. I really want `log_level` to default so we don't have to specify it everywhere, but that doesn't play nicely with rest params. How would you feel about changing `...args` to an array so we can have a default log_level? eg:

```suggestion
module.exports = (args, logLevel=log.level) => new Promise((resolve, reject) => {
```

Then all the callers would pass an array of args rather than a variable number of parameters?

Also, use camel case for variables, eg: `logLevel` not `log_level`.

Reply:
Yes, it is a trade-off, because if we do that we also lost the ability to pass the args as separated parameters which is desired in this kind of functions. I'll do the change.

Reply:
Fixed